RadixWallet/Clients/CardCarouselClient/CardCarouselClient+Interface.swift

```diff
@@ -0,0 +1,32 @@
+// MARK: - CardCarouselClient
+public struct CardCarouselClient: Sendable {
+	public var cards: Cards
+	public var closeCard: CloseCard
+
+	init(
+		cards: @escaping Cards,
+		closeCard: @escaping CloseCard
+	) {
+		self.cards = cards
+		self.closeCard = closeCard
+	}
+}
+
+// MARK: - CarouselCard
+public enum CarouselCard: Hashable, Sendable {
+	case threeSixtyDegrees
```

Comment:
Do we plan on letting the Cards be an enum? Is it because we need to know what to do when the card is pressed, and we dont wanna represent that with a closure? Especially not if this is to be moved into Sargon?

Reply:
It looks like the repertoire of cards is very limited, and they are completely static, so it seems like a good model. The action will be known at compile time, not sure it makes sense to store it on the model, it can just be handled in the reducer, or possibly the client.